regolith/helpers/l_generalhelper.py

```diff
@@ -0,0 +1,84 @@
+"""Helper for finding people's information in the people collection
+"""
+
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    search_collection,
+    collection_str,
+)
+HELPER_TARGET = "lister"
+
+def subparser(subpi):
+    subpi.add_argument(
+        "coll",
+        help='collection where the lister is going to run.')
+    subpi.add_argument(
+        "--kv_filter",
+        nargs="+",
+        help='search the given collection by key element pairs')
+    subpi.add_argument(
+        "-r",
```

Comment:
I think this needs a long name, --return_fields or sthg.  Help could "keys of fields whose values will be printed after filtering."

Reply:
done!